Documentation/development.md

```diff
@@ -0,0 +1,83 @@
+# CMO Development
+
+
+## Run CMO Locally
+### Assumptions
+* Should have a cluster running
+* Has setup Go development env
+
+### Build and Run CMO
+
+The `make` target `run-local` builds the `operator` and runs CMO locally
+
+```shell
+export KUBECONFIG=/path/to/kubeconfig/file
+make run-local
+```
+
+### Fix jsonnet
+
+**TL;DR:**
+1. build operator
+1. use `hack/local-cmo.sh`
+1. modify assets/yamls
+1. backport
+
+
+Sometimes we may not want to rebuild the operator. E.g. you may need
```

Comment:
Do we have examples of such use cases?

Reply:
Yes, we do, I got this information from Pawel and it seems like a common use-case

Say some changes need to go into jsonnet (including vendored bundle pulled from different git repo)
The simpler worflow seems to modify assets directly, make it work and then backport these changes to jssonet (including the vendored jsonnet in a different git repos)

Reply:
In fact it was the reason why we removed a change that I had put into local-run.sh to build `operator` every single time.